data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/databaseenrich/GetGeoIP2Data.java

```diff
@@ -17,23 +36,53 @@
  */
 public class GetGeoIP2Data implements GetGeoData {
 
+    private static final Logger LOG = LoggerFactory.getLogger(GetGeoIP2Data.class);
+    public static final String COUNTRY_NAME = "country_name";
+    public static final String CONTINENT_NAME = "continent_name";
+    public static final String REGION_NAME = "region_name";
+    public static final String CITY_NAME = "city_name";
+    public static final String ORGANIZATION_NAME = "organization_name";
+    public static final String NETWORK = "network";
+    public static final String COUNTRY_ISO_CODE = "country_iso_code";
+    public static final String IP = "ip";
+    public static final String REGION_ISO_CODE = "region_iso_code";
+    public static final String TIMEZONE = "timezone";
+    public static final String LOCATION = "location";
+    public static final String ASN = "asn";
+    public static final String POSTAL = "postal";
+    private DatabaseReader.Builder readerEnterprise;
+    private Country country;
+    private Continent continent;
+    private City city;
+    private Location location;
+    private Subdivision subdivision;
+    private Long asn;
+    private String organizationName;
+    private Network network;
```

Comment:
Where do you get the values for these 3 attributes? Seems missing.

Reply:
Removed unused code.